frontend/components/Domain/Recipe/RecipePage/RecipePageParts/RecipePageEditorToolbar.vue

```diff
@@ -1,28 +1,52 @@
 <template>
-  <div class="d-flex justify-start align-center">
+  <div class="d-flex justify-start align-top py-2">
     <RecipeImageUploadBtn class="my-1" :slug="recipe.slug" @upload="uploadImage" @refresh="imageKey++" />
     <RecipeSettingsMenu
       class="my-1 mx-1"
       :value="recipe.settings"
       :is-owner="recipe.userId == user.id"
       @upload="uploadImage"
     />
+    <v-spacer />
+    <v-container class="py-0" style="width: 40%;">
+      <v-select
+        v-model="recipe.userId"
+        :items="allUsers"
+        item-text="fullName"
+        item-value="id"
+        :label="$tc('general.owner')"
+        hide-details
+      >
+        <template #prepend>
+          <UserAvatar :user-id="recipe.userId" :tooltip="false" />
+        </template>
+      </v-select>
+      <v-card-text v-if="ownerHousehold" class="pa-0 d-flex" style="align-items: flex-end;">
+        <v-spacer />
+        <v-icon>{{ $globals.icons.household }}</v-icon>
+        <span class="pl-1">{{ ownerHousehold.name }}</span>
+      </v-card-text>
+    </v-container>
```

Comment:
Thoughts on limiting this to admins only?

Reply:
That's a good idea, we can limit it to admins or the current owner